### ISSUE TITLE
Add New User Activity Option To Email When File Changed in Starred Directory

### ISSUE THREAD

**Issue by cwmke** [nextcloud/activity#4]
### Desired Feature
1. Go to the **Activity** tab under **Personal** Settings
2. Select option to email if "A new file or folder has been created in a starred directory"
3. Whenever a file or directory withing a starred directory is created, email the user at the specified time interval.

### Server configuration
**Operating system**: Linux

**Web server:** Apache

**Database:** Postgresql

**PHP version:** 7.0.8-1

**Nextcloud version:** Nextcloud 9.0.52 (stable)

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated

**Where did you install Nextcloud from:** source

I have a specific directory I like to be emailed about after changes occur within.  Currently, the only choice I have is to email when **A new file or folder has been created**.  But this applies to all of my files and diretories and quickly obfuscates the changes I wish to see.

I would like to be able to star a directory/directories and be emailed when changes happen only within these starred directories.  This would require adding an additional option under **Activity** in Personal settings.  I belive this could benefit many users with broad use cases.

Thanks for any consideration!

This has been opened as at the request of @MorrisJobke.  It is the same as [issue #353 from server](https://github.com/nextcloud/server/issues/353#issuecomment-231685042).

**Comment by MorrisJobke** [comment]
cc @nickvergessen

**Comment by nickvergessen** [comment]
Yeah currently it's only possible to filter for favorites in the Stream.
Would be nice to have, but it's actually not that simple, especially with subfolders, etc. We would need to mount all storages for each user etc, it's quite messy.

**Comment by DarkknightAK** [comment]
can it be solved with the workflow engine?

If Favorite folder file (mimetyp specific) was added every day an e-mail will be send to user.

The "problem" is that you will have than two different setting areas.
